lib/db/DB.ts

```diff
@@ -31,31 +19,35 @@ class DB {
   public sequelize: Sequelize.Sequelize;
   public models: Models;
 
-  constructor(private config: DBConfig, private logger: Logger) {
-    assert(Number.isInteger(config.port) && config.port > 1023 && config.port < 65536, 'port must be an integer between 1024 and 65535');
-
-    this.sequelize = this.createSequelizeInstance(this.config);
+  /**
+   * @param storage the file path for the sqlite database file, if not specified the db is stored in memory
```

Comment:
Is the database stored in the memory when `storage` is set to `:memory:`, not specified or both?